<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetPosSetting</class>
 <widget class="QWidget" name="WidgetPosSetting">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>165</width>
    <height>440</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Item</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QLabel" name="label_type">
        <property name="text">
         <string>PolygonContrlPoint</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Location[m]</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_x">
          <property name="text">
           <string>x</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_x">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-9999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="label_y">
          <property name="text">
           <string>y</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="customDoubleSpinBox" name="doubleSpinBox_y">
          <property name="decimals">
           <number>3</number>
          </property>
          <property name="minimum">
           <double>-9999.000000000000000</double>
          </property>
          <property name="maximum">
           <double>9999.000000000000000</double>
          </property>
          <property name="singleStep">
           <double>0.010000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>318</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>customDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ModelCustomWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
